src/js/components/RadioButtonGroup/doc.js

```diff
@@ -61,3 +61,22 @@ export const doc = RadioButtonGroup => {
 
   return DocumentedRadioButtonGroup;
 };
+
+export const themeDoc = {
+  'radioButtonGroup.gap': {
+    description: `The amount of spacing between child elements. This
+    should not be used in conjunction with 'wrap' as the gap elements
+    will not wrap gracefully.`,
```

Comment:
Not sure this level of description is relevant for a theme prop, thoughts?

Reply:
its just copied from box, i can remove that section

Reply:
removed that section